docs/fidesops/docs/guides/users.md

```diff
@@ -2,19 +2,18 @@
 
 We include some user-related endpoints for the Fidesops Admin UI. In this section, we'll cover:
 
-- I need permissions to be able to create a user. So how do I create the first user?
-- How do I create other users?
-- How do I log in and log out?
-
+- How to create the first user
+- How to create other users?
+- How to log in and log out?
 
 
 ## Creating the first user
 
-To create the first user, run the following `make` command:
+To create the first user:
 
-`make user`
+Run `make server` in top-level `fidesops` directory, then run `make user`.
```

Comment:
I don't think you need to run `make server` if all you want to do is run `make user`. `make user` itself spins up what you need.